packages/fxa-content-server/webpack.config.js

```diff
@@ -45,11 +45,12 @@ const webpackConfig = {
   },
 
   resolve: {
-    extensions: ['.js'],
+    extensions: ['.ts', '.js'],
     modules: [
       path.resolve(__dirname, 'app/scripts'),
       path.resolve(__dirname, 'app/scripts/templates'),
       path.resolve(__dirname, 'app'),
+      path.resolve(__dirname, '.tscompiles/scripts'),
```

Comment:
Typo here? `s/tscompiles/tscompiled/`

Reply:
Huh, turns out this isn't needed at all!